### PR TITLE
LCWG 37, 38, 39, 40 & 41

### PR DESCRIPTION
**Pull request by nighthawk24**

### CODE REVIEW

File: meeting-notes/lcwg-dev-39.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Light Client Working Group Devs Meeting 39
+### Meeting Date/Time: Thursday, December 1st, 2021 17:00 UTC
+### Meeting Duration: 30 minutes
+### [Video of the meeting](not-recorded)
+### Moderator: Pacu Gindre @pacu
+### Attendees: Adi @nighthawk24, Carter @ccjernigan, Kris @nuttycom, Gygaxin, Bobloblaw, Jack Gavigan, Daniel Wolande
```
Comment (pacu) [inline meeting-notes/lcwg-dev-39.md:6]:
```suggestion
### Attendees: Adi @nighthawk24, Carter @ccjernigan, Kris @nuttycom, Gygaxis, Bobloblaw, Jack Gavigan, Daniel Wolande
```

File: meeting-notes/lcwg-dev-39.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+ - Zingo Labs developers shared the issues around Transaction Builder and Kris responded
+ - Issues around clarity of creating a PR and what the process is, and timeline of feedback.
+ - Identify ways for community members to help with PR triage, reviews and contributions.
+ - Review policy currently is x number of PRs for y type of code for z type of developer. 
+
+https://github.com/zcash/secant-android-wallet/blob/main/.github/pull_request_template.md
+https://github.com/zcash/secant-android-wallet/tree/main/.github/ISSUE_TEMPLATE
+
+ - ZIP 317 support is now available from librustzcash, and has at least been made available to the IOS SDK (I believe the Android side is in progress.) zcashd support is blocked on the completion of https://github.com/zcash/zcash/pull/6217
```
Comment (pacu) [inline meeting-notes/lcwg-dev-39.md:26]:
```suggestion
 - ZIP 317 support is now available from librustzcash, and has at least been made available to the IOS SDK (I believe the Android side is in progress.) but not enabled due to lack of testing. Zcashd support is blocked on the completion of https://github.com/zcash/zcash/pull/6217
```

File: meeting-notes/lcwg-dev-41.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - Bunch of new code in the compose UI, some of that will be ported to the SDK
+* iOS SDK
+  - 0.18-beta is on the way following the fix of file system cache
+  - Main change is the way blocks are downloaded, reduced number of blocks are stored. Once the filesystem has the cache, a migration function to dump the cache.
+  - The built in Filemanager higher level API does not have streaming function, it has save to disk. The improvement is minimal on iOS.
+  - New work on inner changes to accommodate interface changes for DAGSync. The bigger changes are the latest one and the 0.18.0
+  - Dropped support for cocoa pods on v0.18.0
+* Lightwalletd
+  - Shardtree state PR crate is in review.
+  - Stack of changes: we can begin refactoring zcashd - add note commitment tree position to the combat block format, then changes to zcashd to compute root of a subtree every 16-20 commitments (stored in cached db) - needs change of C++ implementation of incremental Merkel tree to the frontier type change, then a full history of 2^16 subtree roots can be made available to lightwalletd.
```
Comment (pacu) [inline meeting-notes/lcwg-dev-41.md:33]:
```suggestion
  - Stack of changes: we can begin refactoring zcashd - add note commitment tree position to the compact block format, then changes to zcashd to compute root of a subtree every 16-20 commitments (stored in cached db) - needs change of C++ implementation of incremental Merkle tree to the frontier type change, then a full history of 2^16 subtree roots can be made available to lightwalletd.
```

File: meeting-notes/lcwg-dev-41.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  - Approx. timeline - next week zcashd part work, then hopefully done by end of Jan, then release zcashd release for end of service height - hoping to get the zcashd changes in place.
```
Comment (pacu) [inline meeting-notes/lcwg-dev-41.md:34]:
```suggestion
  - Approx. timeline - next week zcashd part work, then hopefully done by end of Jan, then release zcashd for end of service height - hoping to get the zcashd changes in place.
```

**Review by pacu — CHANGES_REQUESTED** [review]
Thanks Adi! minor things spotted

**Comment by nighthawk24** [comment]
Thanks for the review!

Resolved all the issues.

**Review by pacu — APPROVED** [review]

**Pull request merged**